doc/user/layouts/shortcodes/views-indexes/table-usage-pattern.html

```diff
@@ -3,6 +3,7 @@
 | Usage Pattern | General Guideline |
 |--------------------------------------------------------------------------------|--------------------|
 | View results are accessed from a single cluster only                                         | View with an [index](/sql/create-index) |
-| View results are accessed across [clusters](/concepts/clusters)                            | Materialized view  |
+| View results are accessed across [clusters](/concepts/clusters)<br>(e.g., queries are served from a cluster different from the cluster that maintains the view results) | Materialized view  |
```

Comment:
https://preview.materialize.com/materialize/32118/concepts/indexes/#indexes-on-views-vs-materialized-views

Reply:
Maybe we can extract the text so the column doesn't get too wide? Or, we could change the whole block to a list. It gets hard to read if the browser doesn't have a whole screen available (or the screen is small).